### PR TITLE
buildpack-deps: Update hera to 0.3.2.

### PR DESCRIPTION
**Pull request by aarlt**
Replaces #10337.

### CODE REVIEW

File: scripts/docker/buildpack-deps/Dockerfile.ubuntu1804 [hunk 1]
```diff
@@ -92,7 +92,7 @@ RUN set -ex; \
 ARG HERA_HASH="622663cb3bc1fa07213c6e1fe8070c5c259096e75039a46d014b2a3448b5cf44"
```
Comment (axic) [inline scripts/docker/buildpack-deps/Dockerfile.ubuntu1804]: need to change this.

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1804]: Yep.. we still need to wait to get the hashes.. I just prepared this already

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1804]: Ah, yes! you're right!

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1804]: Hmm.. the script is currently not verifying `EVMONE_HASH`/`HERA_HASH`

File: scripts/docker/buildpack-deps/Dockerfile.ubuntu1804 [hunk 2]
```diff
@@ -84,20 +84,20 @@ RUN set -ex; \
...
 ARG HERA_MAJOR="0"
 ARG HERA_MINOR="3"
-ARG HERA_MICRO="0"
+ARG HERA_MICRO="2"
 RUN set -ex; \
 	HERA_VERSION="$HERA_MAJOR.$HERA_MINOR.$HERA_MICRO"; \
 	TGZFILE="hera-$HERA_VERSION-linux-x86_64.tar.gz"; \
 	wget https://github.com/ewasm/hera/releases/download/v$HERA_VERSION/$TGZFILE; \
-	sha256sum $TGZFILE; \
+	sha256sum $TGZFILE | grep ${HERA_HASH}; \
```
Comment (axic) [inline scripts/docker/buildpack-deps/Dockerfile.ubuntu1804:100]: Hmm, what was the point of the original line? :)

Comment (aarlt) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1804:100]: Don't know ;)

**Review by cameel — APPROVED** [review]

File: scripts/docker/buildpack-deps/Dockerfile.ubuntu1804 (hunk 2 above)
Comment (axic) [reply scripts/docker/buildpack-deps/Dockerfile.ubuntu1804:100]: I didn't even know that grep returns 0 or 1 exit codes whether it found something or not.

**Review by axic — APPROVED** [review]

**Pull request merged**